### PR TITLE
12.0 mig website_event_type_description

### PR DESCRIPTION
**Pull request by ernestotejeda**
* Adapt views
* readme by fragments

Cc @Tecnativa

### CODE REVIEW

**Comment by rafaelbn** [comment]
Please @ernestotejeda check runbot warnings :red_circle:

File: website_event_type_description/README.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+To edit your event type description from the frontend, you need to:
+
+#. Go to `your events page </event>`_.
+#. Make sure you *Filter by Category* is enabled in the *Customize* menu.
```
Comment (rafaelbn) [inline website_event_type_description/README.rst:43]: @ernestotejeda make this step and you get

500: Internal Server Error

```
Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/addons/base/models/qweb.py", line 346, in _compiled_fn
    return compiled(self, append, new, options, log)
  File "<template>", line 1, in template_536_741
  File "<template>", line 2, in body_call_content_740
  File "<template>", line 79, in foreach_725
KeyError: 'type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/addons/base/models/ir_http.py", line 203, in _dispatch
    result = request.dispatch()
  File "/.repo_requirements/odoo/odoo/http.py", line 833, in dispatch
    r = self._call_function(**self.params)
  File "/.repo_requirements/odoo/odoo/http.py", line 344, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/http.py", line 340, in checked_call
    result.flatten()
  File "/.repo_requirements/odoo/odoo/http.py", line 1268, in flatten
    self.response.append(self.render())
  File "/.repo_requirements/odoo/odoo/http.py", line 1261, in render
    return env["ir.ui.view"].render_template(self.template, self.qcontext)
  File "/.repo_requirements/odoo/odoo/addons/base/models/ir_ui_view.py", line 1284, in render_template
    return self.browse(self.get_view_id(template)).render(values, engine)
  File "/home/odoo/OCB-12.0/addons/website/models/ir_ui_view.py", line 318, in render
    return super(View, self).render(values, engine=engine, minimal_qcontext=minimal_qcontext)
  File "/home/odoo/OCB-12.0/addons/web_editor/models/ir_ui_view.py", line 27, in render
    return super(IrUiView, self).render(values=values, engine=engine, minimal_qcontext=minimal_qcontext)
  File "/.repo_requirements/odoo/odoo/addons/base/models/ir_ui_view.py", line 1293, in render
    return self.env[engine].render(self.id, qcontext)
  File "/.repo_requirements/odoo/odoo/addons/base/models/ir_qweb.py", line 59, in render
    result = super(IrQWeb, self).render(id_or_xml_id, values=values, **context)
  File "/.repo_requirements/odoo/odoo/addons/base/models/qweb.py", line 274, in render
    self.compile(template, options)(self, body.append, values or {})
  File "/.repo_requirements/odoo/odoo/addons/base/models/qweb.py", line 353, in _compiled_fn
    raise QWebException("Error to render compiling AST", e, path, node and etree.tostring(node[0], encoding='unicode'), name)
odoo.addons.base.models.qweb.QWebException: 'type'
Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/addons/base/models/qweb.py", line 346, in _compiled_fn
    return compiled(self, append, new, options, log)
  File "<template>", line 1, in template_536_741
  File "<template>", line 2, in body_call_content_740
  File "<template>", line 79, in foreach_725
KeyError: 'type'

Error to render compiling AST
KeyError: 'type'
Template: 536
Path: /templates/t/t/div/div[2]/div[4]/div[1]/ul[3]/t/li/a
Node: <a t-attf-href="&#10;            /event?{{ keep_query('country', 'date', 'organizer', type=type['type'][0]) }}&#10;        " t-attf-class="nav-link#{searches.get('type') == str(type['event_type_id'] and type['event_type_id'][0]) and ' active' or ''}"><t t-esc="type['event_type_id'][1]"/>
                        <span class="badge badge-pill float-right"><t t-esc="type['event_type_id_count']"/></span>
                    </a>
                
```

**Review by rafaelbn — DISMISSED** [review]

**Comment by pedrobaeza** [comment]
Please rebase now that #147 is fixed

File: website_event_type_description/views/event.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<template id="index" inherit_id="website_event.index">
+    <xpath expr="//div[@class='row mt32 mb32']" position="before">
```
Comment (chienandalu) [inline website_event_type_description/views/event.xml]: use `hasclass()` to avoid warnings

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
Changes done

**Review by chienandalu — APPROVED** [review]
Great. Tested on runbot :+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Although this includes a migration script that technically depends on https://github.com/OCA/openupgradelib/pull/148, I'm merging as this doesn't block new module installations.

**Pull request merged**